tests/support/sockjs_server.js

```diff
@@ -54,7 +54,8 @@ function startServer(port, config, prefix) {
     } else {
       serve(req, res, function(err) {
         var status = err ? err.statusCode : 404;
-        return res.writeHead(status).end();
```

Comment:
`res.writeHead` returns `http.ServerResponse`, so the previous code should be OK. The docs even say:
> Returns a reference to the ServerResponse, so that calls can be chained.

Reply:
should be... but in reality, the test doesn't pass without the change from this PR.

Have you tried?

Reply:
Which specific test(s) are you referring to?

Reply:
Here's an example from https://github.com/sockjs/sockjs-client/pull/534
https://travis-ci.org/github/sockjs/sockjs-client/jobs/761021386#L416

And another from https://github.com/sockjs/sockjs-client/pull/536
https://travis-ci.org/github/sockjs/sockjs-client/jobs/762011841#L431

If you look the builds for some PRs, there are a number of examples.